docs/Getting Started/Setting-Up/part-4.md

```diff
@@ -60,11 +60,11 @@ Your source tree should look like the following:
 |--keystone.js
 ```
 
-## Adding a `addEvent` view
+## Adding an `addEvent` view
 
-First we want our pair of new files to make up the route and the view, these should be `routes/views/addEvent` and `templates/views/addEvent.pug`.
+First we want our pair of new files to make up the route and the view. These should be `routes/views/addEvent` and `templates/views/addEvent.pug`.
 
-As we are not a pug tutorial, here's a page we prepared earlier:
+As this is not a pug tutorial, here's a page we prepared earlier:
 
 ```jade
```

Comment:
Is this GfM formatting still jade, or should it be `pug`?

Reply:
Doesn't look like it's supported yet